Source/CesiumRuntime/Private/CesiumGltfComponent.cpp

```diff
@@ -779,10 +779,26 @@ static void loadPrimitive(
   }
 
   int materialID = primitive.material;
+
   const Material& material =
       materialID >= 0 && materialID < model.materials.size()
           ? model.materials[materialID]
           : defaultMaterial;
+
+  bool isUnlit = material.hasExtension<ExtensionKhrMaterialsUnlit>() &&
+                 !options.pMeshOptions->pNodeOptions->pModelOptions
+                      ->ignoreKhrMaterialsUnlit;
+  if (isUnlit) {
+    Material& unlitMaterial = const_cast<Material&>(material);
+    unlitMaterial.emissiveFactor = {1.0, 1.0, 1.0};
```

Comment:
Why should the unlit material have an emissiveFactor of 1.0? I don't quite understand the logic here.